tests/foreman/cli/test_puppetplugin.py

```diff
@@ -117,11 +117,15 @@ def test_positive_enable_disable_logic(destructive_sat, destructive_caps):
     )
 
     # Disable puppet on Capsule and check it succeeded.
-    result = destructive_caps.execute('foreman-maintain plugin purge-puppet')
+    result = destructive_caps.execute(
+        'foreman-maintain plugin purge-puppet --remove-all-data', timeout='20m'
```

Comment:
Optional: ForemanMaintain robottelo CLI can be implemented for this subcommand and can be used here, with default timeout set to 20m

Reply:
Yes it can, but it's out of scope of this PR. In this quick fix I just wanted to harden the disable case (plus added timeout as the default one was not enough). We can add the CLI support for `plugin` command elsewhere and update this test after that.